Geometry2d/src/Circle.cpp

```diff
@@ -1,6 +1,6 @@
-#include "Circle.hpp"
-#include "Segment.hpp"
-#include <Constants.hpp>
+#include <Geometry2d/Circle.hpp>
+#include <Geometry2d/Segment.hpp>
+#include <rj_constants/constants.hpp>
```

Comment:
Why does this depend on constants? It should really be an agnostic geometry library.

Reply:
It depends on constants because `Circle::hit` uses `RobotRadius`, and I didn't want to change any behavior.

Reply:
WHAT THE HECK.

Sigh. TODO, please?

Reply:
done